pkg/interface/chat/src/js/components/lib/profile-overlay.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { cite } from "/lib/util";
+import { Sigil } from "/components/lib/icons/sigil";
+
+export const OVERLAY_HEIGHT = 250;
+
+export class ProfileOverlay extends Component {
+  constructor() {
+    super();
+
+    this.popoverRef = React.createRef();
+    this.onDocumentClick = this.onDocumentClick.bind(this);
+  }
+
+  componentDidMount() {
+    document.addEventListener("mousedown", this.onDocumentClick);
+    document.addEventListener("touchstart", this.onDocumentClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener("mousedown", this.onDocumentClick);
+    document.removeEventListener("touchstart", this.onDocumentClick);
+  }
+
+  onDocumentClick(event) {
+    const { popoverRef } = this;
+    // Do nothing if clicking ref's element or descendent elements
+    if (!popoverRef.current || popoverRef.current.contains(event.target)) {
+      return;
+    }
+
+    this.props.onDismiss();
+  }
+
+  render() {
+    const { contact, ship, color, topSpace, bottomSpace } = this.props;
+
+    let top, bottom;
+    debugger;
+    if (topSpace < OVERLAY_HEIGHT / 2) {
+      top = `0px`;
+    }
+    if (bottomSpace < OVERLAY_HEIGHT / 2) {
+      bottom = `0px`;
+    }
+    if (!(top || bottom)) {
+      bottom = `-${Math.round(OVERLAY_HEIGHT / 2)}px`;
+    }
+    const containerStyle = { top, bottom, left: "100%" };
+
+    const isOwn = window.ship === ship;
+
+    return (
+      <div
+        ref={this.popoverRef}
+        style={containerStyle}
+        className="flex-col shadow-6 br2 bg-white bg-gray0-d inter absolute z-1 f9 lh-solid"
+      >
+        <div style={{ height: "160px" }}>
+          <Sigil
+            ship={ship}
+            size={160}
+            color={color}
+            classes="brt2"
+            svgClass="brt2"
+          />
+        </div>
+        <div className="pv3 pl3 pr2">
+          {contact && contact.nickname && (
+            <div className="b white-d">{contact.nickname}</div>
+          )}
+          <div className="mono gray2">{cite(`~${ship}`)}</div>
+          {!isOwn && (
+            <Link
+              to={`/~chat/new/dm/~${ship}`}
+              className="b--green0 b--green2-d b--solid bw1 green2 mt3 tc pa2 pointer db"
+            >
+              Send Message
+            </Link>
+          )}
+          {isOwn && (
+            <a
+              href="/~groups/me"
```

Comment:
We don't want to go to our root identity. We want to go to our card in the group in Groups. 

We can see that path from `props.association["group-path"]` in chat.js, for example. If the associated group is `~zod/mygroup` then it's just `~groups/view/~zod/mygroup/zod` (if you haven't shared your card with the group yet, does `view` still render? I think it would.)

If it starts with `/~/` then yeah, go to `/me`.